k8s-deployment/kernelserver.template.yml

```diff
@@ -0,0 +1,113 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: kernelserver-deployment-${appname}-1
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kernelserver-${appname}-1
+  template:
+    metadata:
+      labels:
+        app: kernelserver-${appname}-1
+    spec:
+      containers:
+      - name: kernelserver
+        image: ${image}
+        imagePullPolicy: Always
+        ports:
+        - containerPort: ${svcport} # omsport # 32558
+        - containerPort: ${rmiport} # rmiport # 32556
+        command: ["/root/amino/start-kernelserver.sh"]
+        args: ["oms-svc-${appname}", "${omssvcport}", "${rmiport}", "${svcport}"]
```

Comment:
Shouldn't this be omsrmiport instead of omssvcport ?
I think, kernel server needs oms ip and rmi port to get the oms registry. And omsService is obtained by registry.lookup. right ? In that case omssvcport may be of no use ? Missing something ?

Reply:
Ohk. Got it. My interpretation of omsrmiport and omssvcport was exact opposite of what is used.
We actually have 2 ports. one for host rmi registry and the other for services within the registry(only 1 service in our case). I assumed omsrmiport is registry port and omssvcport is service port. But its vice versa

Reply:
Name change issue has been addressed to avoid confusion.